content/manuals/engine/storage/bind-mounts.md

```diff
@@ -4,91 +4,145 @@ title: Bind mounts
 weight: 20
 keywords: storage, persistence, data persistence, mounts, bind mounts
 aliases:
-- /engine/admin/volumes/bind-mounts/
-- /storage/bind-mounts/
+  - /engine/admin/volumes/bind-mounts/
+  - /storage/bind-mounts/
 ---
 
-Bind mounts have been around since the early days of Docker. Bind mounts have
-limited functionality compared to [volumes](volumes.md). When you use a bind
-mount, a file or directory on the host machine is mounted into a container.
-The file or directory is referenced by its absolute path on the host
-machine. By contrast, when you use a volume, a new directory is created within
-Docker's storage directory on the host machine, and Docker manages that
-directory's contents.
+When you use a bind mount, a file or directory on the host machine is mounted
+from the host into a container. By contrast, when you use a volume, a new
+directory is created within Docker's storage directory on the host machine, and
+Docker manages that directory's contents.
 
-The file or directory does not need to exist on the Docker host already. It is
-created on demand if it does not yet exist. Bind mounts are very performant, but
-they rely on the host machine's filesystem having a specific directory structure
-available. If you are developing new Docker applications, consider using
-[named volumes](volumes.md) instead. You can't use Docker CLI commands to directly
-manage bind mounts.
+## When to use bind mounts
 
-![Bind mounts on the Docker host](images/types-of-mounts-bind.webp?w=450&h=300)
+Bind mounts are appropriate for the following types of use case:
 
-> [!TIP]
->
-> Working with large repositories or monorepos, or with virtual file systems that are no longer scaling with your codebase?
-> Check out [Synchronized file shares](/manuals/desktop/features/synchronized-file-sharing.md). It provides fast and flexible host-to-VM file sharing by enhancing bind mount performance through the use of synchronized filesystem caches.
```

Comment:
Do we still need to refer to the features provided by Docker Desktop? For example, currently we do mention that bind-mounts are created on the daemon host, but it may be relevant to the user that with Docker Desktop we provide extra mechanisms to make that work for _host_ locations (but at a performance penalty for having to bridge / synchronise to the daemon host (VM))

Reply:
OK, I added this:

```diff
diff --git a/content/manuals/engine/storage/bind-mounts.md b/content/manuals/engine/storage/bind-mounts.md
index e00820e90e..4b5d07c09d 100644
--- a/content/manuals/engine/storage/bind-mounts.md
+++ b/content/manuals/engine/storage/bind-mounts.md
@@ -60,6 +60,11 @@ the mount.
   If you're using a remote Docker daemon, you can't create a bind mount to
   access files on the client machine in a container.
 
+  For Docker Desktop, the daemon runs inside a Linux VM, not directly on the
+  native host. Docker Desktop has built-in mechanisms that transparently handle
+  bind mounts, allowing you to share native host filesystem paths with
+  containers running in the virtual machine.
+
 - Containers with bind mounts are strongly tied to the host.
 
   Bind mounts rely on the host machine's filesystem having a specific directory
```

Reply:
Thanks! Yup, that works for me 👍